terraform/browser/browser.tf.json.template.py

```diff
@@ -78,6 +138,45 @@ def emit():
             }
         },
         'resource': {
+            'aws_cloudfront_response_headers_policy': {
+                'browser': {
+                    'name': 'browser',
+                    'security_headers_config': {
+                        # The 'override' argument controls which header value is
+                        # used when the response from the origin also includes
+                        # the same header. True means CloudFront will use the
+                        # value from this policy, False means CloudFront will
+                        # use the value received from the origin.
+                        'content_security_policy': {
+                            'override': True,
+                            'content_security_policy': content_security_policy()
+                        },
+                        'content_type_options': {  # value set to 'nosniff'
```

Comment:
Don't understand this comment. Please explain.

Reply:
Presence of this config adds the `X-Content-Type-Options` header with a value set to `nosniff`. Since the `nosniff` value is not apparent from the config and its options I thought this could use a comment stating the value, however since this described in the Terraform [documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_response_headers_policy#security-headers-config), I've removed the comment here.